Processor.php

```diff
@@ -60,7 +74,26 @@ public function processFile(array $config)
             mkdir($dir, 0755, true);
         }
 
-        file_put_contents($realFile, "# This file is auto-generated during the composer install\n" . Yaml::dump($actualValues, 99));
+        if ('yml' == $fileExtension) {
+            $output = <<<EOF
+# This file is auto-generated during the composer install
+%actual.values%
+EOF
+            ;
+            $output = str_replace('%actual.values%', Yaml::dump($actualValues, 99), $output);
```

Comment:
why using a str_replace here instead of the string concatenation we were using ?